nipype/interfaces/brainsuite.py

```diff
@@ -0,0 +1,980 @@
+    diffusionIterations = traits.Int(3, usedefault=True,
+                                     desc='diffusion iterations',
+                                     argstr='-n %d')
+    edgeDetectionConstant = traits.Float(0.64, usedefault=True,
+                                         desc='edge detection constant',
+                                         argstr='-s %f')
+    radius = traits.Float(1, usedefault=True,
+                          desc='radius of erosion/dilation filter',
+                          argstr='-r %f')
+    dilateFinalMask = traits.Bool(True, usedefault=True,
+                                  desc='dilate final mask', argstr='-p')
+    trim = traits.Bool(True, usedefault=True, desc='trim brainstem',
+                       argstr='--trim')
+    outputMaskFile = File(mandatory=False, desc='save smooth brain mask',
+                          argstr='--mask %s', hash_files=False)
+    outputDiffusionFilter = File(desc='diffusion filter output',
+                                 argstr='--adf %s', hash_files=False)
+    outputEdgeMap = File(desc='edge map output', argstr='--edge %s',
+                         hash_files=False)
+    outputDetailedBrainMask = File(desc='save detailed brain mask',
+                                   argstr='--hires %s', hash_files=False)
+    outputCortexFile = File(desc='cortex file', argstr='--cortex %s',
+                            hash_files=False)
+    verbosityLevel = traits.Float(1, usedefault=True,
+                                  desc=' verbosity level (0=silent)',
+                                  argstr='-v %f')
+    noRotate = traits.Bool(True, usedefault=True,
+                           desc='retain original orientation'
+                           '(default behavior will auto-rotate input NII files'
+                           'to LPI orientation)', argstr='--norotate')
+    timer = traits.Bool(desc='show timing', argstr='--timer')
+
+
+class bseOutputSpec(TraitedSpec):
+    outputMRIVolume = File(desc='path/name of brain-masked MRI volume')
+    outputMaskFile = File(desc='path/name of smooth brain mask')
+    outputDiffusionFilter = File(desc='path/name of diffusion filter output')
+    outputEdgeMap = File(desc='path/name of edge map output')
+    outputDetailedBrainMask = File(desc='path/name of detailed brain mask')
+    outputCortexFile = File(desc='path/name of cortex file')
+
+
+class bse(CommandLine):
+    input_spec = bseInputSpec
+    output_spec = bseOutputSpec
+    _cmd = 'bse'
+
+    def _list_outputs(self):
+
+        outputs = self.output_spec().get()
+        if isdefined(self.inputs.outputMRIVolume):
+            outputs['outputMRIVolume'] = (os.getcwd() + '/' +
+                                          self.inputs.outputMRIVolume)
+        else:
+            outputs['outputMRIVolume'] = self._gen_filename('outputMRIVolume')
+
+        if isdefined(self.inputs.outputMaskFile):
```

Comment:
these almost seem like a pattern and i would like to simplify their creation. we can work on this after next week.